src/Plugin/Show/Payload.php

```diff
@@ -257,7 +261,7 @@ public static function hasMatch(Request $request): bool {
 	 */
 	public function getHandlerClassName(): string {
 		return __NAMESPACE__ . '\\' . match (static::$type) {
-			'full tables' => 'FullTablesHandler',
+			'expanded tables' => 'ExpandedTablesHandler',
```

Comment:
It's much better here to use for every field is ::class linking, for example:
```suggestion
			'expanded tables' => ExpandedTablesHandler::class,
```
So it will be more easier in future to find what and where used, without magic strings.

Reply:
Yes, I agree.